app/models/user.rb

```diff
@@ -16,6 +16,10 @@ class User < ActiveRecord::Base
   has_many :granted_policies, class_name: "Policy", foreign_key: "granter_id"
 
   after_create :grant_implicit_policy
+  
+  def timeout_in
+    5.minutes
```

Comment:
If you need more info on how `Settings` are computed, we are using the [rails_config](https://github.com/railsconfig/config) gem for this.